index.html

```diff
@@ -11,74 +11,132 @@
 </head>
 
 <body>
-  <header class="top-container">
-    <nav class="top-nav">
+  <nav class="top-nav">
+    <div class="wrapper">

```

Comment:
generalnie wszystkie sekcje wymagaly malego refactoru - bo mieliscie tak, ze kazda sekcja miala osobne style - mimo ze wygladaly podobnie w jakis sposob - np szerkosc tresci w srodku. 

zrobilem uniwersalna klase `.wrapper` 
ktora powinna miec tylko taki css:
```
.wrapper {
   margin-left: auto;
   margin-right: auto;
   width: 90%;
}
``` 

Co nam to daje - a no, ustawiasz szerokosc strony raz, i zapominasz o tym - jak trzeba zmienic, to w jednym miejscu a nie we wszystkich sekcjach osobno - DRY (Dont Repeat Yourself), oraz latwiej to zmieni pozniej w media queries - tez zmieniasz tylko klase wrapper a nie wszystkie sekcje

Reply:
Każda sekcja miała oddzielne style, bo każdy robił inną ;P